odoo/addons/base/data/res_country_data.xml

```diff
@@ -261,13 +261,13 @@
             <field eval="236" name="phone_code" />
         </record>
         <record id="cd" model="res.country">
-            <field name="name">Democratic Republic of the Congo</field>
+            <field name="name">Congo-Kinshasa</field>
             <field name="code">cd</field>
             <field name="currency_id" ref="CDF" />
             <field eval="243" name="phone_code" />
         </record>
         <record id="cg" model="res.country">
-            <field name="name">Congo</field>
+            <field name="name">Congo-Brazzaville</field>
```

Comment:
I have checked the iso standard (and also on google maps), can you use official names?
Also in the task comments, there is no concensus to use what you suggest.

https://www.iso.org/obp/ui/#iso:code:3166:CD - Democratic Republic of the Congo
https://www.iso.org/obp/ui/#iso:code:3166:CG - Republic of the Congo